salt/files/cloud.providers.d/ec2.conf

```diff
@@ -2,13 +2,13 @@
 {% set cloud = salt['pillar.get']('salt:cloud', {}) -%}
 ec2_ubuntu_public:
   minion:
-    master: {{ cloud['master'] }}
+    master: {{ cloud.get('master', 'salt') }}
```

Comment:
It would be really cool if someone eventually ripped out this type of stuff and used the salt_settings with it's default merging (while making sure the defaults were correct or course :smile: I didn't put all the defaults into defaults.yaml). But not specifically a requirement here.